### DISCUSSION TITLE
Update codeowners-related docs after landing Bitbucket codeowner file support?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#29511] [Suggest an Idea]
### Tell us more.

This discussion is needed after this PR landed a new feature, but did _not_ update the docs:

- #29502

## Things to discuss and then fix

Can a maintainer please tell me what to do for these items?

- [x] [`expandCodeOwnersGroups`](https://docs.renovatebot.com/configuration-options/#expandcodeownersgroups) lists only `gitlab` as supported platform, does that need to be updated?
- [x] [`assigneesFromCodeOwners`](https://docs.renovatebot.com/configuration-options/#assigneesfromcodeowners) does _not_ list supported platforms in the table at all! It mentions GitHub and GitLab docs though... Should I add the [Bitbucket Code owners docs](https://confluence.atlassian.com/bitbucketserver/code-owners-1296171116.html) as well?
    > See [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) or [GitLab](https://docs.gitlab.com/ee/user/project/code_owners.html) documentation for details on syntax and possible file locations.
- [x] Same problem as for `assigneesFromCodeOwners` applies to the [`reviewersFromCodeOwners`](https://docs.renovatebot.com/configuration-options/#reviewersfromcodeowners). Maybe this should mention supported platforms in the table, and link to the bitbucket docs too?

**Comment by rarkins** [comment]
expandCodeOwnerGroups is still gitlab-only

assigneesFromCodeowners is not platform-specific, because CODEOWNERS can be in the root of any platform repository

**Reply by HonkingGoose** [reply]
@rarkins want me to _only_ add a link to the Bitbucket docs about the Codeowners name and location then?

**Reply by rarkins** [reply]
Do we have a corresponding doc already for github and gitlab? If not them I'm not sure we need to state it

**Reply by HonkingGoose** [reply]
Yes, we have corresponding docs for `assigneesFromCodeOwners` _and_ `reviewersFromCodeOwners`, they both have this line:

> See [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) or [GitLab](https://docs.gitlab.com/ee/user/project/code_owners.html) documentation for details on syntax and possible file locations.

**Comment by HonkingGoose** [comment]
I'm resolving this discussion, as I'm improving the docs with this PR:

- #29516
